utils/generate-command-code.py

```diff
@@ -398,6 +444,10 @@ def create_command(name, desc):
         subcommand.group = command.group
         command.subcommands.append(subcommand)
 
+print("Checking all commands...")
+for command in commands.values():
+    check_command_key_specs(command)
```

Comment:
maybe we want to collect the error count, and fail the script if there are errors (otherwise they might go unnoticed)

Reply:
my thought was that if we get an error, it will do a exit(1) instead of a return, something like that
now write it like this so that I can print them all (do a copy)

i can do a error counter or just do a exit (fix it one by one), which one do you prefer?

Reply:
i prefer to print all errors and exit later.